### PR TITLE
[WIP] Onboarding flow

### PR DESCRIPTION
**Pull request by marcinbodnar**

### CODE REVIEW

**Comment by charleslavon** [comment]
This is looking really good @marcinbodnar.  I'm testing through the flows now and will add feedback/questions here

**Comment by charleslavon** [comment]
@marcinbodnar are we intentionally showing these three action buttons? 
Screen Shot 2023-04-24 at 9 28 24 AM

**Comment by charleslavon** [comment]
Nice work @marcinbodnar! This looks so good and works really well for an initial onboarding flow. I think we're in a good spot to get this merged and make iterative improvements such as the ones I mentioned above.

**Comment by heycorwin** [comment]
Sharing feedback and issues around the latest version:

- After "Try it Now" I see a page where the initial Sandbox welcome screen and Editor UI are stacked

Sandbox

- Ideally, we would direct the user straight into the explainer screen, rather than give them options at this point. Is there a reason why we are showing all these options when coming from the "Try it Now" CTA on the Homepage?

- Feedback about the Sandbox UI in general: Because of the variety of component styles and states as well as the fact that they are fairly underdeveloped and retain much of their original styling, it's difficult for the user to discern what is changing, what is interactive, and what is disabled. This makes it hard to follow certain parts of the flow. For example, when the user first forks the component, it's hard to notice what has changes since much of the UI is in a disabled state, including the components tab/header

- Asking the user to "Confirm Code Paste" that they've pasted code by clicking a button in the tooltip feels a bit clunky. It's likely a smoother experience to detect that they've pasted, then prompt them to reload their preview with the next tooltip, rather than having them click a button in between.

Screenshot 2023-04-24 at 10 00 27 AM

- Confirmations / success messages are best positioned in close proximity to the action which has just taken place. In the following case, since we are summarizing what has just changed, it's best to position the tooltip next to the render preview.

Screenshot 2023-04-24 at 9 58 51 AM

- Similarly, it's best to position the next tooltip in close proximity to the search input label. This tooltip also has an issue where the copy is split by the code sample and "Copy" button which disrupts legibility.

Screenshot 2023-04-24 at 9 59 03 AM

- As mentioned previously, it would be better to detect that the user has queried a search rather than have them manually confirm that they have done this by pressing a button in each tooltip to move on.

- When searching for the component, it is unclear which result is the correct one, since it appears that there are many with the same name. We should either automatically rename a forked component, or prompt the user to rename their component when forking, or alternatively, when publishing.

Screenshot 2023-04-24 at 9 59 52 AM

- Component search results have a button to "Embed", but the tooltip reads "Copy Embed Code". We should update either so that they are consistent and so that the user understands that they are meant to click the "Embed" button to copy the embed code as instructed in the tooltip.

- If possible, the "Back" action within each tooltip should use a different styling than the primary CTA so as to reduce it's emphasis. Consider using a gray button with white text.

**Review by charleslavon — APPROVED** [review]
:shipit:

**Comment by AustinBaggio** [comment]
Hey team, created [a doc to organize and prioritize all of the feedback](https://docs.google.com/document/d/1oVujuhFjC1VUsVZPuqMq2UUhVAnNesZ_LYgjcR9IAnw/edit) from here and slack. Please feel free to add to the "Unprioritized" Section as new feedback comes in. 

Great work getting this to where it is. I think we just have to address the "Must do" copy updates and then merge this in. Everything else can be incremental.
